src/Service.Tests/Multidab-config.PostgreSql.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json",
+  "data-source": {
+    "database-type": "postgresql",
+    "connection-string": "Host=localhost;Database=datagatewaytest;username=REPLACEME;password=REPLACEME"
```

Comment:
I dont see the `files` property in any of the config files, how are we specifying the children configs for the tests?

Reply:
If you see runtimeConfigLoader tests, I test with different files as the "root" file and add the files property within that.